src/config/serverConfigDefaults.ts

```diff
@@ -61,6 +61,9 @@ const ServerConfigDefaults: Partial<IServerConfig> = {
     skin_query_max_tree_depth: '3',
     skin_right_nav_show_data_sets: true,
     skin_right_nav_show_examples: true,
+    skin_right_nav_show_installations: true,
+    installations_url: 'http://localhost:3006',
```

Comment:
My bet is that we don't need to make this configurable. The same url should work for everyone.

Reply:
I just removed the two lines and hardcoded installations_url inside RightBar.tsx and PortalHeader.tsx.